include/mxnet/mem_mgr.h

```diff
@@ -1,11 +1,9 @@
-#ifndef MXNET_MEM_MGR_H_
-#define MXNET_MEM_MGR_H_
```

Comment:
Why did you remove these lines? All header files should have this ifndef-define pattern.